src/skip-to-content/skip-to-content.scss

```
@@ -0,0 +1,31 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-skip-to-content {
+  @include iui-blur($opacity: 3);
+  @include iui-focus($offset: 2px, $thickness: 2px);
+  @include themed {
+    color: t(iui-color-foreground-accessory);
+  }
+
+  border-radius: $iui-border-radius-huge;
+  text-decoration: none;
+  padding: ($iui-baseline * 0.75) $iui-m;
+  position: absolute;
+  text-align: center;
+  left: 50%;
+  transform: translateX(-50%) translateY(-170%);
+  transition: background-color $iui-speed-fast ease-in-out, transform $iui-speed-slow ease-in-out,
```

You will need to use a media query to only transition the transform if the user doesn't `prefers-reduced-motion. Same thing in the :focus rule

---

```suggestion
transition: background-color $iui-speed-fast ease-in-out, box-shadow $iui-speed-fast ease-in-out;
@media (prefers-reduced-motion: no-preference) {
	transition: background-color $iui-speed-fast ease-in-out, box-shadow $iui-speed-fast ease-in-out, transform $iui-speed-slow ease-in-out;
}
```
This makes it so if the user does not have a "reduced motion" preference set in their operating system, they get all 3 transition effects.  But if the user does have reduced motion turned on, they only see background-color and box-shadow transition, no motion (from transform).

---

Most other components have the media query for all types of transitions, including background color and box shadow.

---

yes that is a mistake, the media query doesn't make sense for non-motion transitions